ktor-client/ktor-client-core/common/src/io/ktor/client/statement/HttpStatement.kt

```diff
@@ -117,6 +123,19 @@ class HttpStatement(
         }
     }
 
+    /**
+     * Check that all request configuration related to client capabilities have correspondent features installed.
+     */
+    private fun checkCapabilities() {
+        builder.attributes.getOrNull(ENGINE_CAPABILITIES_KEY)?.keys
+            ?.filter { it is HttpClientFeature<*, *> }
+            ?.forEach {
+                requireNotNull(client.feature(it as HttpClientFeature<*, *>)) {
+                    "Consider install $it feature because request requires it to be installed"
```

Comment:
Consider installing $it feature because the request requires it

Reply:
Thanks, done.